Homework_25/script.js

```diff
@@ -0,0 +1,44 @@
+function isObjectEmpty(obj) {
+  if (Object.keys(obj).length == 0) {
+    console.log('empty');
+    return true;
+} else {
+  console.log("isn't empty");
+}
+}
+
+let obj = {};
+
+isObjectEmpty(obj);
+
+let user = {
+  name: "Stas",
+  age: 22,
+  sayHello: function () {
+    alert(`Привіт я ${this.name} мені ${this.age} років`);
+  },
+};
+
+user.sayHello();
+
+let Calc = {
```

Comment:
from capital letter in JS could be only name of Classes or Function Construction

but not primitive variables
so that's a big mistake